django_ai_assistant/models.py

```diff
@@ -1,12 +1,27 @@
 import json
+import uuid
+from typing import Any
 
 from django.conf import settings
 from django.db import models
 from django.db.models import F, Index, Manager
 
+from django_ai_assistant.exceptions import AIAssistantPrimaryKeyTypeNotInListError
+
+
+def get_id_field():
```

Comment:
We don't need that if we do this: https://github.com/vintasoftware/django-ai-assistant/pull/110/files#r1648900685